### PR TITLE
Added health check command

### PR DESCRIPTION
**Pull request by mmoreram**
* Returns 0 if is healthy. Otherwise, returns 1

### CODE REVIEW

File: Console/CheckHealthCommand.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+/**
+ * Class CheckHealthCommand.
+ */
+class CheckHealthCommand extends CommandWithBusAndGodToken
+{
+    /**
+     * Configures the current command.
+     */
+    protected function configure()
```
Comment (cv65kr) [inline Console/CheckHealthCommand.php]: `void` return typehint

Comment (cv65kr) [inline Console/CheckHealthCommand.php]: `void` return typehint

File: Console/CheckHealthCommand.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        return 'Check health';
+    }
+
+    /**
+     * Dispatch domain event.
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return mixed
```
Comment (cv65kr) [inline Console/CheckHealthCommand.php]: `@return mixed` => `@return int`

File: Console/CheckHealthCommand.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+     */
+    protected function dispatchDomainEvent(InputInterface $input, OutputInterface $output)
```
Comment (cv65kr) [inline Console/CheckHealthCommand.php]: `int` return typehint

File: Console/CommandWithBusAndGodToken.php [hunk 4]
```diff
@@ -104,6 +104,7 @@ protected function createGodToken(string $appId): Token
     protected function execute(InputInterface $input, OutputInterface $output)
```
Comment (cv65kr) [inline Console/CommandWithBusAndGodToken.php]: `int` return typehint

File: Console/CheckHealthCommand.php (hunk 3 above)
Comment (mmoreram) [reply Console/CheckHealthCommand.php]: Can't here, because implements an interface :)

File: Console/CheckHealthCommand.php (hunk 1 above)
Comment (mmoreram) [reply Console/CheckHealthCommand.php]: Can't because overwrites a method

File: Console/CheckHealthCommand.php (hunk 2 above)
Comment (mmoreram) [reply Console/CheckHealthCommand.php]: This is an implementation of an interface. Changing this has no sense, I think...

File: Console/CommandWithBusAndGodToken.php (hunk 4 above)
Comment (mmoreram) [reply Console/CommandWithBusAndGodToken.php]: `execute` method is part of the Symfony Console command definition. Can't be changed the signature

**Comment by mmoreram** [comment]
Commands return 0 when everything is fine, or >0 when error

**Comment by cv65kr** [comment]
ok, them

**Pull request merged**